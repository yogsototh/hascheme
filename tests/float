3.14
Float 3.14
