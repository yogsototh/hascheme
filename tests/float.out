Float 3.14
